css/main.css

```diff
@@ -34,6 +34,70 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type="number"] {
+	max-width: 7em;
+
+	/* Remove spinner controls from Firefox */
+	appearance: textfield;
+}
+
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type="number"]:hover,
+input[type="number"]:focus {
+	appearance: number-input;
```

Comment:
Once again: this should not be in `main.css` if you want to prevent a 200-lines, unreadable CSS to appear. This is a cross-browser adjustment that should reside in its own file.